src/v2/Apps/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box, Text } from "@artsy/palette"
+
+export const SavedSearchAlertsApp: React.FC = ({ children }) => {
+  return (
+    <Box>
+      <Text variant="xl" my={4}>
+        Your Alerts
+      </Text>
+      {children}
+    </Box>
+  )
+}
```

Comment:
You also need to use `MetaTags` component here so that the Cypress test passes. For example, [like this](https://github.com/artsy/force/blob/main/src/v2/Apps/_About2/AboutApp.tsx#L16-L20)

Reply:
oh thanks! Good to know!